### PR TITLE
DOCS: Reintroducing APT and YUM installation methods - porting #14492 to master

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Porting: https://github.com/openvinotoolkit/openvino/pull/14492/

Reintroducing APT and YUM installation methods in Linux.

### CODE REVIEW

**Review by kblaszczak-intel — APPROVED** [review]

File: docs/install_guides/installing-openvino-apt.md [hunk 1]
```diff
@@ -1,10 +1,224 @@
...
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
+> **NOTE**: From the 2022.1 release, OpenVINO™ Development Tools can be installed via PyPI only. See [Install OpenVINO Development Tools](#installing-openvino-development-tools) for more information.
 
-The other installation methods are temporarily unavailable.
\ No newline at end of file
+> **IMPORTANT**: By downloading and using this container and the included software, you agree to the terms and conditions of the [software license agreements](https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf).
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:7]:
```suggestion
```
Removing it as it's obsolete.

File: docs/install_guides/installing-openvino-apt.md [hunk 2]
```diff
@@ -1,10 +1,224 @@
...
+  Processor graphics are not included in all processors.
+  See `Product Specifications`_ for information about your processor.
+
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:39]:
```suggestion
  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
```

File: docs/install_guides/installing-model-dev-tools.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ OpenVINO Development Tools is a set of utilities that make it easy to develop an
 
 The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.
 
-In both cases, Python 3.6 - 3.10 need be installed on your machine before starting.
+In both cases, Python 3.7 - 3.10 need be installed on your machine before starting.
```
Comment (sgolebiewski-intel) [inline docs/install_guides/installing-model-dev-tools.md:13]:
```suggestion
In both cases, Python 3.7 - 3.10 need to be installed on your machine before starting.
```

Comment (kblaszczak-intel) [reply docs/install_guides/installing-model-dev-tools.md:13]: I would even make it "needs to be" because just one version is required.

**Review by xu-yuan1 — APPROVED** [review]

**Pull request merged**